### DISCUSSION TITLE
Is there a way to know how much my repository uses Github's storage?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22852] [New to GitHub]
<p>I’d like to know how much data size my repository is in combination with all the branches and the total commit history.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Yes there is. Just check here: <a href="https://github.com/settings/repositories" rel="nofollow noopener">https://github.com/settings/repositories</a> .</p>

**Reply by s-Sayan** [reply]
And How to know how much storage storage I have left?

**Reply by mpboom** [reply]
@s-Sayan, please consult https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage for that
